src/net/sourceforge/kolmafia/webui/RelayLoader.java

```diff
@@ -36,22 +37,40 @@ protected RelayLoader(final String location, final boolean isRelayLocation) {
     }
   }
 
+  private void pauseAndWaitForRelayAction(int timeInMilliseconds, boolean waitForStart) {
+    // Pause for input value of seconds, waking up to check relay server
```

Comment:
waitForStart is always true in this code but I have an uncommitted test that, if I keep, used the same routine to  wait for shutdown.  Not sure it's a good test but the parameter here defers the discussion.